Assets/HoloToolkit/Utilities/Scripts/TextToSpeechManager.cs

```diff
@@ -324,5 +326,54 @@ public void SpeakText(string text)
             LogSpeech(text);
             #endif
         }
+
+        /// <summary>
+        /// RE for stripping HTML
+        /// </summary>
+        //static Regex _htmlRegex = new Regex("<.*?>", RegexOptions.SingleLine);
```

Comment:
Yes, it should. I left it in here becuase it took me a long time to find an RE that actually worked but in the end I plumped for the brute force method as it would allow a progression to encoding in SSML instead of text.